pinot-controller/src/main/java/org/apache/pinot/controller/helix/ControllerRequestURLBuilder.java

```diff
@@ -53,6 +55,10 @@ public String forInstance(String instanceName) {
     return StringUtil.join("/", _baseUrl, "instances", instanceName);
   }
 
+  public String forInstanceUpdateTags(String instanceName, List<String> tags) {
+    return StringUtil.join("/", _baseUrl, "instances", instanceName, "updateTags?tags=" + Joiner.on(",").join(tags));
```

Comment:
Suggest using StringUtils to join
```suggestion
    return StringUtil.join("/", _baseUrl, "instances", instanceName, "updateTags?tags=" + StringUtils.join(tags, ',');
```